### DISCUSSION TITLE
When does the new version gets reflected on vcpkgs.io?

### DISCUSSION THREAD

**Discussion by harshavardhana** [microsoft/vcpkg#26299] [Q&A]
Let's say a new update to a library is merged, how much time does it take to reflect on the vcpkgs.io?

I see that we updated the minio-cpp with v0.1.0 but this has not been reflected yet and it's been more than 4-5 days.

Thanks for your time..

**Comment by autoantwort** [comment]
When somebody thinks it is the right time to manually update the site

**Reply by harshavardhana** [reply]
> When somebody thinks it is the right time to manually update the site

Shouldn't this be automated?

**Reply by autoantwort** [reply]
It should, but they haven't done it for more than a year.
